ogusa/get_micro_data.py

```diff
@@ -66,13 +69,16 @@ def get_calculator(baseline, calculator_start_year, reform=None, data=None, weig
 
     # this increment_year function extrapolates all PUF variables to the next year
     # so this step takes the calculator to the start_year
-    for i in range(calculator_start_year-2013):
+    while calc1.current_year < calculator_start_year:
         calc1.increment_year()
 
+    # running all the functions and calculates taxes
+    calc1.calc_all()
+
     return calc1
 
 
-def get_data(baseline=False, start_year=2016, reform={}):
+def get_data(baseline=False, start_year=2016, reform={}, data=None):
```

Comment:
We might make the default `start_year` be 2018 at this point.

Reply:
Thanks, this is fixed in the latest commits.